src/main/java/io/icker/factions/api/persistents/Faction.java

```diff
@@ -45,6 +45,10 @@ public class Faction {
     @Field("Open")
     private boolean open;
 
+    /** Whether friendly fire is enabled for this faction (null = use global config) */
+    @Field("FriendlyFire")
+    private Boolean friendlyFire;
```

Comment:
Instead of using a boxed boolean type, this should use an enum with either true, false, or use config. I don't think our serialization system can handle this correctly as is and it would be more readable.